src/Dropdown.js

```diff
@@ -119,22 +120,46 @@ export default class Dropdown extends Component {
 
   render() {
     const { handleMouseEnter, handleMouseLeave, handleClick } = this
-    const { mode, children, component } = this.props
+    const { mode, children, component, link } = this.props
     const { isOpen } = this.state
-    const className = cx(this.props.className, {
+    const className = cx('uk-button uk-button-default', {
       'uk-open': isOpen,
     })
-    const DropdownProps = {
+
+    const dropdownClassName = cx('uk-dropdown uk-dropdown-bottom-left', {
+      'uk-open': isOpen,
+    })
+
+    const labelProps = {
+      'aria-expanded': isOpen,
+      'aria-haspopup': true,
+      className: link !== 'navbar' ? className : '',
+      onClick: handleClick,
+      onMouseEnter: mode === 'hover' && handleMouseEnter,
+      onMouseLeave: mode === 'hover' && handleMouseLeave,
+      children: this.props.dropdownLabel || mode,
```

Comment:
This is the label for `button`. 
![screen shot 2017-03-20 at 15 28 13](https://cloud.githubusercontent.com/assets/2022919/24104190/dd31f452-0d81-11e7-9fa0-4aa627a0ffe5.png) 

So, if user does not define the `dropdownLabel`, it will just use `mode` text. The button will say `hover` or `click` on the label.